examples/example.js

```diff
@@ -19,7 +19,6 @@ const opts = {
 
 fastify
   .get('/', opts, function (req, reply) {
-    reply.header('Content-Type', 'application/json').code(200)
```

Comment:
Why this line is removed?

Reply:
it was not needed, those are the defaults.